src/foundry/foundry.js/clientDocuments/scene.d.ts

```diff
@@ -8,9 +18,450 @@ declare global {
    * @see {@link documents.Scenes}            The world-level collection of Scene documents
    * @see {@link applications.SceneConfig}    The Scene configuration application
    *
-   * @param data - Initial data provided to construct the Scene document
    */
-  class Scene extends ClientDocumentMixin(foundry.documents.BaseScene) {}
+  class Scene extends ClientDocumentMixin(foundry.documents.BaseScene) {
+    /**
+     * @param data - Initial data provided to construct the Scene document
+     */
+    constructor(
+      data: ConstructorParameters<typeof foundry.documents.BaseScene>[0],
+      context?: ConstructorParameters<typeof foundry.documents.BaseScene>[1]
+    );
+
+    /**
+     * Determine the canvas dimensions this Scene would occupy, if rendered
+     * @defaultValue `{}`
+     */
+    dimensions: ReturnType<typeof Canvas.getDimensions> | {};
+
+    /**
+     * Track whether the scene is the active view
+     */
+    protected _view: this['data']['active'];
+
+    /**
+     * Track the viewed position of each scene (while in memory only, not persisted)
+     * When switching back to a previously viewed scene, we can automatically pan to the previous position.
+     * @remarks This is intentional public because it is used in Canvas._initializeCanvasPosition() and Canvas.pan()
+     */
+    _viewPosition: { x: number; y: number; scale: number } | {};
+
+    /**
+     * A convenience accessor for whether the Scene is currently active
+     */
+    get active(): this['data']['active'];
+
+    /**
+     * A convenience accessor for the background image of the Scene
+     */
+    get img(): this['data']['img'];
+
+    /**
+     * A convenience accessor for whether the Scene is currently viewed
+     */
+    get isView(): boolean;
+
+    /**
+     * A reference to the JournalEntry entity associated with this Scene, or null
+     */
+    get journal(): InstanceType<ConfiguredDocumentClass<typeof JournalEntry>> | null;
+
+    /**
+     * A reference to the Playlist entity for this Scene, or null
+     */
+    get playlist(): InstanceType<ConfiguredDocumentClass<typeof Playlist>> | null;
+
+    // TODO: Use PlaylistSound instead of BasePlaylistSound
+    /**
+     * A reference to the PlaylistSound document which should automatically play for this Scene, if any
+     */
+    get playlistSound(): InstanceType<ConfiguredDocumentClass<typeof foundry.documents.BasePlaylistSound>> | null;
```

Comment:
The type of this is incorrectly documented as `   * @type {Playlist|null}` by foundry, consider creating an issue.